docs/filestructure.rst

```diff
@@ -0,0 +1,228 @@
+.. _filestructure:
+
+==================================
+File structure example & first app
+==================================
+
+We will deal here with a suggestion how you could structure your files and folders for your individual atk4 project.
+Let's assume you are planning to create at least several pages with some models and views. This example can be expanded
+and modified to your needs and shows just one concept of how to setup an atk4 project.
+
+
+File structure example
+======================
+
+This file structure is a recommendation and no must. It is a best practice example.
+Feel free to experiment with it and find the ideal file structure for your project.
+
+* config
+
+    * db.php
+
+* public_html
+
+    * vendor (contains all needed composer modules - don't touch them)
+
+        * atk4
+
+            * ...
+
+        * autoload.php
+
+        * ...
+
+    * projectfolder
+
+        * forms
+
+            * exampleForm1.php
+
+            * exampleForm2.php
+
+        * models
+
+            * exampleClass1.php
+
+            * exampleClass2.php
+
+        * views
+
+            * View1.php
+
+            * View2.php
+
+        * images
+
+        * index.php
+
+        * init.php
```

Comment:
init.php can be called bootstrap.php and can load autoload.php + load config and set the chdir(__DIR__)

Reply:
I called it init.php, because it's called so in the demos of atk4...and again: init.php is in my opinion clearer than bootstrap.php (which also could mean the framework).